prepare_oim/roles/deploy_containers/common/tasks/set_vip.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+---
+
+- name: Set virtual IP as admin IP if OIM high availability is enabled
+  when:
+    - hostvars['localhost']['oim_ha_data'].ansible_facts.oim_ha.enable_oim_ha | default(false)
+    - hostvars['localhost']['oim_ha_data'].ansible_facts.oim_ha.virtual_ip_address is defined
```

Comment:
can we set a variable for vip_status and use across the playbook. virtual_ip_address  length > 1 also required